helm/templates/config.yaml

```diff
@@ -53,11 +53,7 @@ data:
   SUBSYSTEM_OPENTELEMETRY__SAMPLER_TYPE: {{ ((.Values.opentelemetry_zaakafhandelcomponent).sampler_type) | default "-off" }}
   SUBSYSTEM_OPENTELEMETRY__ENDPOINT: {{ ((.Values.opentelemetry_zaakafhandelcomponent).endpoint) | default (printf "http://%s-opentelemetry-collector:4317" .Release.Name) }}
   {{- end }}
-  {{- if .Values.smartDocuments.enabled }}
-  SMARTDOCUMENTS_ENABLED: true
```

Comment:
This one hasn't yet been applied to TEST
@infojohn should we wait to see if the current solution works?